website/docs/Pipelines/Exome_Germline_Single_Sample_Pipeline/exome.methods.md

```diff
@@ -0,0 +1,31 @@
+
+QC metrics (base distribution by cycle, insert size metrics, mean quality by cycle, and quality score distribution) were collected for the aligned, unsorted read-groups using Picard CollectMultipleMetrics. The read-group specific aligned BAMs were then aggregated and duplicate reads were flagged using Picard MarkDuplicates assuming queryname-sorted order and the parameter --OPTICAL_DUPLICATE_PIXEL_DISTANCE=2500, which is appropriate for patterned flowcells.
+
+The aggregated BAM file was then sorted using Picard SortSam with coordinate sort order. The fingerprints of separate read groups were verified using Picard CrosscheckFingerprints with a LOD threshold of -10. Cross-sample contamination was checked using verifyBamID2.
+
+The aligned BAM was then scattered for parallelization during base recalibration. A Base Quality Score Recalibration (BQSR) table was created with GATK BaseRecalibrator using the original base qualities (under the OQ Sam tag). The model was applied using ApplyBQSR with the static-quantized-quals option used according to the [Functional Equivalence specification](https://github.com/CCDG/Pipeline-Standardization/blob/master/PipelineStandard.md) ([Regier et al., 2018](https://www.nature.com/articles/s41467-018-06159-4)). Recalibrated BAM files were then merged using Picard GatherSortedBamFiles.
+
+QC metrics were calculated for the base-recalibrated BAM using interval and baits lists with Picard CollectHsMetrics. Fingerprints were verified using Picard CheckFingerprint and high duplication levels and chimerism were checked using the calculated summary metrics.
+
+The final base-recalibrated BAM was converted to CRAM using Samtools view and validated using Picard ValidateSamFile.
+
+### Variant Calling
+
+Prior to variant calling, the variant calling interval list was subsetted to enable parallelization. Variant calling was performed using GATK HaplotypeCaller with the annotation parameters -G StandardAnnotation -G StandardHCAnnotation. Genome quality (GQ) bands were set using  -GQB 10 -GQB 20 -GQB 30 -GQB 40 -GQB 50 -GQB 60 -GQB 70 -GQB 80 -GQB 90. The resulting GVCFs were merged using Picard MergeVcfs and the final VCF file was validated using GATK ValidateVariants. Variant metrics were calculated using Picard CollectVariantCallingMetrics.
```

Comment:
I would say "split" instead of "subsetted"
I though exomes also got AS_StandardAnnotation?
It might be worth specifying "Reference block genotype quality (GQ) bands" -- though if you do that, we should probably update the WGS version too

Reply:
`AS_StandardAnnotation` is included (or excluded) based on the value of the input variable `make_gvcf` which is defaulted to true. As far as I see, this is always defaulted to true and would have to be overwritten in the inputs file to be set to false. Which is a very long way of saying, yes, I believe `AS_StandardAnnotation` is included for exomes by default

Reply:
Added the changes. For the WGS pipeline which uses the Haplotypecaller for GATK 3.5, it doesn't seem like the variant discovery WDL (https://github.com/broadinstitute/warp/blob/master/tasks/broad/GermlineVariantDiscovery.wdl), it uses those GQ bands. I might be missing it.

Reply:
We probably haven't updated WGS, which we should do, but that's not your problem, Liz.